src/posthog-surveys.ts

```diff
@@ -168,9 +167,12 @@ export class PostHogSurveys {
                     ? this.instance.featureFlags.isFeatureEnabled(survey.targeting_flag_key)
                     : true
 
-                const eventBasedTargetingFlagCheck = hasEvents(survey) ? activatedSurveys?.includes(survey.id) : true
+                const eventBasedTargetingFlagCheck = assignableWindow.__PosthogExtensions__.hasEvents(survey)
+                    ? activatedSurveys?.includes(survey.id)
+                    : true
 
-                const overrideInternalTargetingFlagCheck = canActivateRepeatedly(survey)
+                const overrideInternalTargetingFlagCheck =
+                    assignableWindow.__PosthogExtensions__.canActivateRepeatedly(survey)
```

Comment:
this is the meat of the improvement

these few lines save 6% bundle size by not depending on preact until after the extensions have loaded, maybe it should be moved into the load script so it's clear they're being loaded from the er scsript